snc.sh

```diff
@@ -31,7 +31,7 @@ BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
 CRC_PV_DIR="/mnt/pv-data"
 SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc"
 SCP="scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i id_ecdsa_crc"
-MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp-dev-preview}
+MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
```

Comment:
Why?

Reply:
Because now rc bits of 4.15 available and it is from `https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp` instead `https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp-dev-preview`

Reply:
This makes reviewing a lot smoother/easier if this is included in the commit log from the start, no need to go back and forth in comments